database/migrations/20240715163245-initial-db.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Do we need this file? My understanding is that we shouldn't

Reply:
Alright I will remive this file.